contrib/win32/win32compat/wmain_sshd.c

```diff
@@ -151,6 +151,50 @@ prereq_setup()
 		free(info);
 }
 
+/*
+* 1) Create %programdata%\ssh - Administrator group(F), system(F), authorized users(RX).
+* 2) Create %programdata%\ssh\logs - Administrator group(F), system(F)
+* 3) copy <binary_location>\sshd_config_default to %programdata%\ssh\sshd_config
+*/
+static void 
+create_prgdata_ssh_folder()
+{
+	/* create ssh cfg folder */
+	char ssh_cfg_dir[PATH_MAX] = { 0, };
+	strcpy_s(ssh_cfg_dir, _countof(ssh_cfg_dir), get_program_data_path());
+	strcat_s(ssh_cfg_dir, _countof(ssh_cfg_dir), "\\ssh");
+	if (!create_directory_withsddl(ssh_cfg_dir, "O:BAD:PAI(A;OICI;FA;;;SY)(A;OICI;FA;;;BA)(A;OICI;0x1200a9;;;AU)"))
```

Comment:
how about non-admin user? have read permission?

Reply:
yes. this is inline with directory permissions on Unix. Everyone has read permissions to this folder.

Reply:
AU is authenticated user. We should grant WD Read & Execute, right?

Reply:
write is given only to administrator groups, system..
all other users (Authenticated users) will have Read & Execute permissions.

Are we missing anything here?